### ISSUE TITLE
Form elements do not have associated labels for event subscriptions page

### ISSUE THREAD

**Issue by LeoAndTheTree** [CMU-313/Mayan-EDMS#132]
The current Accessibility Score for the event subscriptions page is 85. One of the issues with this page is that the form elements do not have labels, such that assistive devices cannot read our the form.
<img width="977" alt="Screen Shot 2021-09-09 at 6 22 38 PM" src="https://user-images.githubusercontent.com/22224479/132770364-73ad79e5-61e7-4fc4-888f-58ceb9c53200.png">
